.travis.yml

```diff
@@ -11,6 +11,11 @@ addons:
 cache:
  - apt
 
+sudo: required
```

Comment:
It is required to use `sudo required`, if you use travis-ci and docker: From the docker notes: [Setup Docker with travis-ci](https://docs.travis-ci.com/user/docker/)

> There are a few additions you need to make to your .travis.yml to have your builds be routed to our new stack:

> sudo: required
services: - docker